src/transports/http.rs

```diff
@@ -65,22 +65,22 @@ impl Transport for Http {
 /// Will execute synchronously when first polled.
 pub struct FetchTask {
   id: usize,
-	url: String,
-	client: Arc<reqwest::Client>,
+  url: String,
+  client: Arc<reqwest::Client>,
   request: String,
 }
 
 impl Future for FetchTask {
-	type Item = rpc::Value;
-	type Error = RpcError;
```

Comment:
haha, sorry about messing up the indentation here :)

(do you think we could switch to 4spaces at some point? 2 is a bit tight)